bitbots_vision/src/bitbots_vision/dynamic_color_space.py

```diff
@@ -138,8 +138,8 @@ def image_callback(self, image_msg):
 
         # Drops old images
         image_age = rospy.get_rostime() - image_msg.header.stamp
-        if 0.1 < image_age.to_sec() < 1000.0:
-            rospy.loginfo('Vision: Dropped incoming Image-message, because its too old!')
+        if 1.0 < image_age.to_sec() < 1000.0:
+            rospy.logwarn_throttle(2, 'Vision: Dropped incoming Image-message, because its too old! ({} sec)'.format(image_age.to_sec()))
```

Comment:
Is this related to the PR?

Reply:
No not really, I was wondering why this showed up here. It is from another pull request yet merged and closed pull request.